compose/service.py

```diff
@@ -249,6 +249,19 @@ def create_container(self,
 
         return Container.create(self.client, **container_options)
 
+    def exec_create(self, command, **kwargs):
+        try:
+            container = self.get_container(number=kwargs["index"])
+        except KeyError:
+            container = self.get_container()
+        return Container.create_exec(self.client, container.id, command, **kwargs)
+
+    def exec_start(self, exec_id, **kwargs):
+        return Container.start_exec(self.client, exec_id, **kwargs)
+
+    def get_exec_exit_code(self, exec_id):
+        return self.client.exec_inspect(exec_id).get("ExitCode")
```

Comment:
This is the feedback I was looking for since I'm a bit loss in `compose`'s logic.